sdk/requirements/python-3-8.txt

```diff
@@ -9,4 +9,5 @@ pydantic
 plotly
 pyyaml
 great_expectations
-ruamel.yaml
\ No newline at end of file
+ruamel.yaml
```

Comment:
Why are we using both ruamel.yaml and pyyaml? Can you make a low-hanging fruit task on the backlog to unify them?

Reply:
https://linear.app/aqueducthq/issue/ENG-1569/unify-the-use-of-pyyaml-and-ruamelyaml-packages-in-the-sdk